src/index.html

```diff
@@ -22,6 +22,390 @@
+      >
+        <div class="logo">
+          <img
+            src="./images/imac.jpeg"
+            alt="logo_link"
+          />
+        </div>
+
+        <div class="text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="code">Product code: 195434</div>
+
+        <div class="stars_4">
+          <div class="stars_active"></div>
+          <div class="stars_active"></div>
+          <div class="stars_active"></div>
+          <div class="stars_active"></div>
+          <div class="stars_gr"></div>
+          <span class="review">Reviews: 5</span>
+        </div>
+
+        <div class="price">
+          <span class="azer">Price:</span>
+          $2,199
+        </div>
+        <button
+          class="button"
+          data-qa="card-hover"
+        >
+          <span class="by">BUY</span>
+        </button>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="logo">
+          <img
+            src="./images/imac.jpeg"
+            alt="logo_link"
+          />
+        </div>
+
+        <div class="text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="code">Product code: 195434</div>
+
+        <div class="stars_4">
+          <div class="stars_active"></div>
+          <div class="stars_active"></div>
+          <div class="stars_active"></div>
+          <div class="stars_active"></div>
+          <div class="stars_gr"></div>
+          <span class="review">Reviews: 5</span>
+        </div>
+
+        <div class="price">
+          <span class="azer">Price:</span>
+          $2,199
+        </div>
+        <button
+          class="button"
+          data-qa="hover"
+        >
+          <span class="by">BUY</span>
+        </button>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="logo">
+          <img
+            src="./images/imac.jpeg"
+            alt="logo_link"
+          />
+        </div>
+
+        <div class="text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="code">Product code: 195434</div>
+
+        <div class="stars_4">
+          <div class="stars_active"></div>
+          <div class="stars_active"></div>
+          <div class="stars_active"></div>
+          <div class="stars_active"></div>
+          <div class="stars_gr"></div>
+          <span class="review">Reviews: 5</span>
+        </div>
+
+        <div class="price">
+          <span class="azer">Price:</span>
+          $2,199
+        </div>
+        <button
+          class="button"
+          data-qa="hover"
+        >
+          <span class="by">BUY</span>
+        </button>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="logo">
+          <img
+            src="./images/imac.jpeg"
+            alt="logo_link"
```

Comment:
The 'alt' attribute for the product image is set to 'logo_link', which is not descriptive. Consider using a more descriptive 'alt' text that accurately describes the image content, such as 'iMac 27 Retina 5K'.